mix.exs

```diff
@@ -47,6 +47,8 @@ defmodule PrimaAuth0Ex.MixProject do
 
   defp dev_deps do
     [
+      {:absinthe, "~> 1.6", optional: true},
+      {:absinthe_plug, "~> 1.5", optional: true},
```

Comment:
These are not `dev_deps`.

I would mark even `plug` as an optional dependency (probably also `redix`, since the caching is optional, but in such a case we should change the default to false).

Reply:
You're right, I fixed the dependencies!

`redix` could be optional too, but I would leave it for a future PR